blog_system/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path, include
+from debug_toolbar.toolbar import debug_toolbar_urls
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("blog/", include("blog.urls", namespace="blog")),
+    path("", include("django.contrib.auth.urls")),
+] + debug_toolbar_urls()
```

Comment:
Here, you're trying to concatenate `debug_toolbar_urls()` to your `urlpatterns`. As mentioned in the previous comment, this is not the correct way to include Django Debug Toolbar in your URLs. Please use the correct import and path as suggested.